manuscript/chapter3-contributor-mode.md

```diff
@@ -150,7 +150,12 @@ The big issue with chats is to stay on topic, and get back to the topic when the
 
 An example of this is when one piles many questions/issues in one single message, when people join after some time the discussion,they are unable to catch up from the very top sometimes. In that case is better to face one thing at the time and pass on to the next when the previous is exhausted and in doing that there is the need to be intransigent with the discussion remaining on topic.
 
-E-mails allow for recap, or a single notification for a group of mails, chat systems send a notification for every and each message that gets to be posted, this resulting in a dancing and ringing smartphone in less than a minute.
+E-mails allow for recap, or a single notification for a group of mails, chat systems send a notification for every and each message that gets to be posted, this resulting in a dancing and ringing smartphone in less than a minute.  
+
+For this reasons a chat solution is fine for some stuff and not for everything. Also understand the kind of technology for chatting is important, is not just a flow of messages like IRC (that anyway is getting abandoned for various reasons), but include usability, multi-device usage and various levels of anonymous.  
```

Comment:
*various levels of anonymity

Reply:
Let me take the opportunity and refer you to https://fosdem.org/2020/schedule/event/nextgencontributors/ which explained, which tool (chat, forum, mailing list etc) is optimal for kind of requirements.

Reply:
I saw that talk but for me there was to much blabla so I preferred to not add there but maybe for other people is better.